### PR TITLE
[DW-1633] Add News Hub

### PR DESCRIPTION
**Pull request by JodiWarren**
- Create News page
- Rename Digital Security Homepage doctype to Generic Homepage
- Add white background styles
- Add consistent spacing between newshub items
- Set news to `/news` in the content

Please note: There may be some manual configuration needed in the console at `/hst:hst/hst:configurations/common-preview/hst:workspace/hst:containers/news-hub`. Make sure it matches the configuration at `/hst:hst/hst:configurations/common/hst:workspace/hst:containers/news-hub`, including creating the nodes if necessary.

### CODE REVIEW

File: repository-data/application/src/main/resources/hcm-config/namespaces/website.cnd [hunk 1]
```diff
@@ -3,25 +3,19 @@
...
-[website:financial] > hippostd:relaxed, hippotranslation:translated, website:basedocument
-  orderable
-
 [website:aboutapi] > hippo:compound, hippostd:relaxed
   orderable
 
 [website:address] > hippo:compound, hippostd:relaxed
   orderable
 
-[website:hubnewsandevents] > hippostd:relaxed, hippotaxonomy:classifiable, hippotranslation:translated, website:basedocument
```
Comment (ShaneAhern-IRE) [inline repository-data/application/src/main/resources/hcm-config/namespaces/website.cnd:23]: Please do not remove any entries within the CND.
Any documents already created with that document type will break

Comment (JodiWarren) [reply repository-data/application/src/main/resources/hcm-config/namespaces/website.cnd:23]: :thumbsup: Not quite sure how that happened but great catch.

Comment (JodiWarren) [reply repository-data/application/src/main/resources/hcm-config/namespaces/website.cnd:23]: @ShaneAhern-IRE I'm not able to revert this removal without causing a 503. Is there a good way of trying to debug that?

**Comment by JodiWarren** [comment]
These changes were re-implemented in https://github.com/NHS-digital-website/hippo/pull/1075

**Pull request closed without merging**